projects/storefrontlib/shared/components/chat-messaging/avatar/avatar.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ICON_TYPE } from '@spartacus/storefront';
+import { MessageEvent } from '../messaging';
+
+@Component({
+  selector: 'cx-avatar',
+  templateUrl: './avatar.component.html',
+})
+export class AvatarComponent implements OnInit {
+  @Input() event: MessageEvent;
```

Comment:
I suggest changing the model name.  Quotes feature will use this component and they do not know the concept of Event
```suggestion
  @Input() message: Message;
```

Reply:
Yup, seems better.